### PR TITLE
add get project for a single project

### PR DESCRIPTION
**Pull request by maleck13**
@wtrocki  @luigizuccarelli  @PhilipGough  @mmurphy added a 

```
get project --project=<guid> 

```
command. Take a look let me know what you think

### CODE REVIEW

File: commands/get/projects.go [hunk 1]
```diff
@@ -24,17 +24,17 @@ type projectsCmd struct {
 }
 
 //Project Defines our cli command including its flags and usage then returns the command to allow a user to do specific operations on projects
-func (pc *projectsCmd) Project() cli.Command {
+func (pc *projectsCmd) Projects() cli.Command {
```
Comment (maleck13) [inline commands/get/projects.go]: these are just refactoring to have it called projects rather than project

File: commands/types.go [hunk 2]
```diff
@@ -2,7 +2,33 @@ package commands
 
 // define all the types used in a common file
 
+// Project defines the project response object
 type Project struct {
-	Title string `json:"title"`
-	Guid  string `json:"guid"`
+	AuthorEmail    string      `json:"authorEmail"`
```
Comment (maleck13) [inline commands/types.go]: really handy tool for creating these structs from existing json

https://github.com/ChimeraCoder/gojson

File: commands/get/project.go [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+	"io/ioutil"
+	"net/http"
+	"text/template"
+
+	"github.com/feedhenry/rhm/commands"
+	"github.com/feedhenry/rhm/storage"
+	"github.com/urfave/cli"
+)
+
+var projectTemplate = `
```
Comment (wtrocki) [inline commands/get/project.go]: Maybe we can have separate task to create `OutputFormatter` that would present output depending on the format user requesting by specifying `output` option. For example values can be: 
- `json` - raw json option, 
- `yaml` - we can use json2yaml converter, 
- `table` - we can use one of this libs
https://github.com/tatsushid/go-prettytable or https://github.com/crackcomm/go-clitable

What you think about it? Can we create new ticket?

Comment (maleck13) [inline commands/get/project.go]: Yeah I was thinking this too 👍 
https://github.com/feedhenry/rhm/issues/17
Feel free to modify it as it on mentions json

**Comment by luigizuccarelli** [comment]
@maleck13 - I really like the Makefile addition

**Comment by maleck13** [comment]
@luigizuccarelli  yeah its handy. Nice for running the ci tests and also for doing a release. Ok gonna merge so that everyone can pick up the make file and ci changes

**Pull request merged**